A32NX/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -286,4 +309,81 @@ class A32NX_FWC {
         this.ldgMemo = showInApproach || invalidRadioMemo || this.flightPhase === 8 || this.flightPhase === 7;
         SimVar.SetSimVarValue("L:A32NX_FWC_LDGMEMO", "Bool", this.ldgMemo);
     }
+
+    altitudeWarning(_deltaTime) {
+        const indicatedAltitude = Simplane.getAltitude();
+        const reset = SimVar.GetSimVarValue("L:A32NX_ALT_DEVIATION_SHORT", "Bool");
+        if (reset = true) {
```

Comment:
This is an assignment to a constant. Either use comparison `===` or:
```suggestion
        if (reset) {
```